wx-core/src/generated.rs

```diff
@@ -488,6 +488,21 @@ impl<const OWNED: bool> ListBoxMethods for CheckListBoxIsOwned<OWNED> {
         }
     }
 }
+impl<const OWNED: bool> From<CheckListBoxIsOwned<OWNED>> for ListBoxIsOwned<OWNED> {
+    fn from(o: CheckListBoxIsOwned<OWNED>) -> Self {
+        unsafe { ListBoxIsOwned::from_ptr(o.as_ptr()) }
+    }
+}
+impl<const OWNED: bool> TryFrom<&ListBoxIsOwned<OWNED>> for CheckListBoxIsOwned<false> {
```

Comment:
Generic TryFrom can't be implemented because of conflict with blanket impl in core.

We need repeat this per MxN combination of Base x Derived.